src/types/index.js

```diff
@@ -58,4 +59,25 @@ export type AllMarkdownRemark = {
   }[]
 };
 
+export type GatsbyImage = {
+  node: {
+    childImageSharp: typeof gatsbyImageData;
+    name?: string;
+  }
+}
+
+export type GatsbyImages = GatsbyImage[];
+
+export type ServiceData = {
+  node: {
+    frontmatter: {
+      description: string,
+      name: string,
+      logo: {
+        childImageSharp: typeof gatsbyImageData;
+      }
+    }
+  }
+}
```

Comment:
Move to Service

Reply:
ServiceData type is used in other files than services.js, so it has to be exported anyway.
Should it be exported from services.js ? or can we leave it there

Reply:
Leave it there, thanks.